image/tarexport/load.go

```diff
@@ -116,7 +119,9 @@ func (l *tarexporter) Load(inTar io.ReadCloser, outStream io.Writer, quiet bool)
 		if err != nil {
 			return err
 		}
-		imageIDsStr += fmt.Sprintf("Loaded image ID: %s\n", imgID)
+		imageIDsStr.WriteString("Loaded image ID: ")
+		imageIDsStr.WriteString(imgID)
```

Comment:
> Looks like this needs to be converted to a string here;
> 
> ```go
> 		imageIDsStr.WriteString(imgID.String())
> ```



Sure, don't know how I missed that.


> 
> That said, I guess this could also just concatenate the strings (to avoid the `Sprintf`);
> 
> ```go
> imageIDsStr.WriteString("Loaded image ID: " + imgID.String() + "\n")
> ```
> 
> A quick bench between using the code in this PR, and `imageIDsStr.WriteString("Loaded image ID: " + imgID.String() + "\n")`;
> 
> ```
> BenchmarkWithBuffer-8   	25752823	        58.00 ns/op	      89 B/op	       0 allocs/op
> BenchmarkWithConcat-8   	31360424	        37.99 ns/op	      46 B/op	       0 allocs/op
> ```

It seems Go optimizes string concatination. Maybe it allocates a buffer of exact size unlike `bytes.Buffer`.